scripts/wiki/ddl.sql

```diff
@@ -0,0 +1,55 @@
+-- DDL for parsed Wiki data.
+
+CREATE TABLE entities (
+    id TEXT PRIMARY KEY NOT NULL,
+    -- This could be normalized. Not worth it at the moment though, since they aren't used.
+    claims TEXT
+);
+
+-- The FTS5 virtual table implementation doesn't allow for indices, so we rely on ROWID to match entities.
+-- This isn't great, but with a controlled data ingestion setup this allows for stable matching.
+-- Same for foreign keys.
+CREATE VIRTUAL TABLE entities_texts USING fts5(
+    entity_id UNINDEXED,
+    name,
+    description,
+    label
+);
+
+CREATE TABLE articles (
+    entity_id TEXT PRIMARY KEY NOT NULL,
```

Comment:
This is probably just my ignorance about Wikidata, but it seems surprising to me that the article referring to an entity has its own entity ID - perhaps a note would make it a bit easier to understand?